Corona-Warn-App/src/main/res/layout/home_statistics_cards_sevendayrvalue_layout.xml

```diff
@@ -57,16 +58,33 @@
             app:layout_constraintTop_toTopOf="@id/primary_value" />
 
         <TextView
-            android:id="@+id/secondary_value_label"
+            android:id="@+id/secondary_label"
             style="@style/StatisticsCardValueLabel"
             android:layout_width="0dp"
             android:layout_height="wrap_content"
             android:layout_marginHorizontal="@dimen/spacing_normal"
-            android:layout_marginTop="@dimen/spacing_normal"
+            android:layout_marginVertical="@dimen/spacing_normal"
```

Comment:
We should maybe remove some margin/padding so that there is more space for text. Many people will use increased font-sizes on their devices.

![Screenshot from 2021-01-18 18-51-54](https://user-images.githubusercontent.com/1439229/104949065-a281b580-59be-11eb-9c73-0351e601e504.png)

Reply:
I reduced some of the margins